ts/models/conversation.ts

```diff
@@ -1478,17 +1480,24 @@ export class ConversationModel extends Backbone.Model<ConversationAttributes> {
     }
 
     if (details.admins && isArray(details.admins)) {
```

Comment:
This code has a lot of duplication. Could we make it a private method that processes either type and returns what is needed?

Reply:
amended